.changeset/chip.md

```diff
@@ -0,0 +1,17 @@
+---
+'@leafygreen-ui/chip': major
+---
+
+[LG-4121](https://jira.mongodb.org/browse/LG-4121): Removes `popoverZIndex` prop because the `InlineDefinition` component instance will now render in the top layer
```

Comment:
nit: Could avoid a major change by just deprecating the prop, and having it not do anything. Not a huge deal though since we have so many other major changes